### DISCUSSION TITLE
Doesn't work with Synology Firewall on

### DISCUSSION THREAD

**Discussion by hunterwinterton** [nextcloud/all-in-one#1017] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Turn on Synology Firewall

### Expected behavior 
Should be able to load Nextcloud
### Actual behavior 
Nothing happens, page doesn't load even though I added all the ports to the firewall

### Host OS 
Synology DSM

#### Nextcloud AIO version 
1.7.0
#### Current channel 
latest
#### Other valuable info 
<img width="789" alt="Screen Shot 2022-08-12 at 11 55 41 PM" src="https://user-images.githubusercontent.com/68767432/184470941-a77f1440-064c-41b7-8121-c2a0419b308f.png">

**Comment by szaimen** [comment, marked as answer]
I fear I am not really able to help here as I do not have a Synology NAS available. However you may have a look if you have a rule in place that completely drops all connections for rules that are not specified here which should be dropped or you need to really specify rules even for all internal communications which is not desireable for docker networks...

**Comment by szaimen** [comment]
And for the record: this is most likely not an AIO problem but a problem with your firewall and docker networks in general...
